src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -66,7 +66,7 @@ protected function configure(): void
 
         $this
             ->setDefinition([
-                new InputArgument('receivers', InputArgument::IS_ARRAY, 'Names of the receivers/transports to consume in order of priority', $defaultReceiverName ? [$defaultReceiverName] : []),
+                new InputArgument('receivers', InputArgument::IS_ARRAY, 'Names or patterns of the receivers/transports to consume in order of priority', $defaultReceiverName ? [$defaultReceiverName] : []),
```

Comment:
If a pattern matches multiple names, how the order is infered ? By definition order in the ?

Reply:
I was just commenting to be sure of the order with the new pattern matching
This seems ok to me :)